crates/uv/tests/init.rs

```diff
@@ -1423,7 +1423,7 @@ fn init_virtual_project() -> Result<()> {
         dependencies = []
 
         [project.scripts]
-        hello = "foo:hello"
+        foo = "foo:main"
```

Comment:
Maybe I'm mistaken here, but I think this test shows that `uv init --virtual` is already broken (it doesn't create a virtual project in any sense?)

Reply:
Yeah this is fishy — `--virtual` is legacy now but shouldn't be doing this.